README.md

```diff
@@ -128,68 +139,65 @@ optim = CPUOffloadOptimizer(model.parameters(), torch.optim.AdamW, fused=True)
+Install the stable release (recommended):
+```bash
+pip install torchao
+```
 
-## Custom Kernels
+Other options:
+```bash
+# Nightly build
+pip install --pre torchao --index-url https://download.pytorch.org/whl/nightly/cu124
 
-We've added support for authoring and releasing [custom ops](./torchao/csrc/) that do not graph break with `torch.compile()` so if you love writing kernels but hate packaging them so they work all operating systems and cuda versions, we'd love to accept contributions for your custom ops. We have a few examples you can follow
+# Different CUDA versions
+pip install torchao --index-url https://download.pytorch.org/whl/cu118  # CUDA 11.8
+pip install torchao --index-url https://download.pytorch.org/whl/cpu    # CPU only
 
-1. [fp6](torchao/dtypes/floatx) for 2x faster inference over fp16 with an easy to use API `quantize_(model, fpx_weight_only(3, 2))`
-2. [2:4 Sparse Marlin GEMM](https://github.com/pytorch/ao/pull/733) 2x speedups for FP16xINT4 kernels even at batch sizes up to 256
-3. [int4 tinygemm unpacker](https://github.com/pytorch/ao/pull/415) which makes it easier to switch quantized backends for inference
+```
 
-If you believe there's other CUDA kernels we should be taking a closer look at please leave a comment on [this issue](https://github.com/pytorch/ao/issues/697)
+### Development Install
+```
+USE_CPP=0 python setup.py develop  # Skip C++/CUDA extensions
+```
 
+## Composability
+`torch.compile`: A key design principle for us is composability - any custom dtype or memory layout should work with our compiler. We enable kernel implementations in PyTorch, CUDA, C++, or Triton. This allows researchers and engineers to start with high-level dtype and layout logic in pure PyTorch, then progressively optimize performance by implementing lower-level kernels as needed, while maintaining compatibility with the compile infrastructure.
 
-## Alpha features
+[FSDP2](https://github.com/pytorch/torchtitan/blob/main/docs/fsdp.md): Historically most quantization has been done for inference, there is now a thriving area of research combining distributed algorithms and quantization.
 
-Things we're excited about but need more time to cook in the oven
+The best example we have combining the composability of lower bit dtype with compile and fsdp is [NF4](torchao/dtypes/nf4tensor.py) which we used to implement the [QLoRA](https://www.youtube.com/watch?v=UvRl4ansfCg) algorithm. So if you're doing research at the intersection of this area we'd love to hear from you.
 
-1. [MX](torchao/prototype/mx_formats) training and inference support with tensors using the [OCP MX spec](https://www.opencompute.org/documents/ocp-microscaling-formats-mx-v1-0-spec-final-pdf) data types, which can be described as groupwise scaled float8/float6/float4/int8, with the scales being constrained to powers of two. This work is prototype as the hardware support is not available yet.
```

Comment:
it would be nice to keep mx and int8 quantized training callouts as we do plan to bring them out of prototype

Reply:
Lets have a prototype Readme and once they are moved out of prototype lets update the readme?

Reply:
I think a user should be able to text search the main README.md and know where to go for both MX training/inference and int8 training.  Can we keep short references in the main readme please, with links to more info?  Moving the more descriptive sections to a prototype readme sounds fine.

> Lets have a prototype Readme and once they are moved out of prototype lets update the readme?

Can this be in this PR, instead of deleting?

Reply:
if we just have someting like 

Prototype folder contains upcoming features such as MX training and inference (link) and int8 quantized training (link), and more.

that would sgtm.